sequencer/src/block2.rs

```diff
@@ -161,6 +161,8 @@ impl QueryablePayload for BlockPayload {
         // This correctness proof requires a range of its own, which we read into `tx_table_range_proof_[start|end]`.
         //
         // Edge case--the first transaction: tx payload range `start` is implicitly 0 and we do not include this item in the correctness proof.
+        //
+        // TODO why isn't cargo fmt wrapping these comments?
```

Comment:
It doesn't do that. I use an editor extension for this.

Reply:
hmm. It does in jellyfish. But perhaps it's some weird interaction between that repo and one of the many vscode extensions I have.

Reply:
Jellyfish uses rustfmt from nightly and has a rustformat config file.

https://github.com/EspressoSystems/jellyfish/blob/main/flake.nix#L104
https://github.com/EspressoSystems/jellyfish/blob/main/rustfmt.toml